open_mastr/soap_api/metadata/mastr_datapackage.json

```diff
@@ -9,15 +9,29 @@
     ],
     "subject": [
         {
-            "name": null,
-            "path": null
+            "name": "power plant",
+            "path": "http://openenergy-platform.org/ontology/oeo/OEO_00000031"
```

Comment:
What is the function of ontology references?

Reply:
The purpose is that the `@context` field makes some fields in the oemetadata searchable via sparql queries. Amongst others `subject`, `isAbout`, `valueReference`. It makes the metadata JSON to a JSON-LD.
The idea is that in the future datasets are annotated ontologically for two use-cases:
1. **common vocabulary to search for data in the energy domain in a search engine that builds upon linked open data**
Here the prototype of a search engine from LOD-GEOSS project: http://moss.tools.dbpedia.org/search
The usability and visualistion are going to be improved.
When you search for `hub height`, select the concept from the suggestions and search for it on the energy databus, you'll find wind power plant datasets that contain information about hub height you might use for your calculations. The datasets are registered in the databus, which is a metadata catalog that points to decentrally hosted databases. So the data is not actually hosted and maintained on the databus but at individual institutions with the idea to improve data findability in the domain (only works if sufficient datapoints with good metadata participate).
2. **inference of knowlegde with annotated datasets**
simplistic example: I annote the mastr dataset with `power plant` and you know generally what the concept power plant is, but not what types of power plants exist in the energy domain. You could gain knowledge from the information stored via the hierarchical relations in the ontology.
https://openenergy-platform.org/viewer/oeo/ if you search for `power plant` here.